### ISSUE TITLE
Queue metrics in Pipeline Viewer UI - more info required

### ISSUE THREAD

**Issue by marcinpm** [elastic/logstash#10480]
The documentation of the [Pipeline Viewer UI](https://www.elastic.co/guide/en/logstash/6.6/logstash-pipeline-viewer.html) has a picture of Queue section and it says `Queue metrics not available`. However, the documentation doesn't say anything about the Queue itself, let alone about the process of enabling it. 
Perhaps we could add more information that this gets populated when the Persistent Queue is turned on and that's not the in memory queue?

**Comment by marcinpm** [comment]
cc @karenzone

**Comment by marcinpm** [comment]
It seems like there's further context to this one. At the moment there is no way to enable the queue monitoring in the UI. The message `Queue metrics not available` is hard-coded and this functionality is not implemented yet. [Here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/queue.js) is the respective part of the code.
I believe this should highlighted in the documentation as it may mislead the users.
The related ER is [here](https://github.com/elastic/kibana/issues/26749).

**Comment by roaksoax** [comment]
I agree that the current state is misleading users. So what I think should happen is:

- [ ] Decide whether to remove the 'queue' section altogether or change the message to be less ambiguous
- [ ] Update the documentation to reflect that
- [ ] Provide queue metrics in the future.

Thoughts?

**Comment by willemdh** [comment]
Also just noticed we have "Queue metrics not available" for Logstash pipeline monitoring.  So what is needed to make the queue metric availbale in Logstash monitoring UI? it is not clear at all when reading through https://www.elastic.co/guide/en/logstash/current/logstash-pipeline-viewer.html
